src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public Figure getRandomFigure(){

```

Comment:
According to the checklist, you should generate random values in the `FigureSupplier` class instead of inside the figure classes. Make sure to implement the logic for generating random figures here.